package.json

```diff
@@ -50,6 +50,7 @@
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
```

Comment:
```suggestion
    "@typescript-eslint/parser": "^5.30.7",
```

Reply:
bumping anything is beyond the scope of this pr.